ten_quick_tips_live_coding.md

```diff
@@ -105,8 +105,8 @@ Standing makes the experience more interactive and less monotonous,
 and draws the learners' attention away from their screens to you,
 which helps getting the point you are making across.
 
-It helps to have a high table/standing desk or lectern.
-[FIXME: explain]
+It helps to have a high table/standing desk or lectern
+so you can have your laptop at a comfortable height for typing.
```

Comment:
Standing also encourages you to look at your audience rather than your screen: if you are sitting, you are likely to do what you normally do when you are sitting, which is look at your own computer.

Reply:
+1